doc/classes/EditorTranslationParserPlugin.xml

```diff
@@ -20,6 +23,17 @@
 		func get_recognized_extensions():
 		    return ["csv"]
 		[/codeblock]
+		[b]Note:[/b] If you override parsing logic for standard script types (gd, C# etc.), it would be better to load the [code]path[/code] argument using [method ResourceLoader.load]. This is because built-in scripts are loaded as [Resource] type, not [File] type.
+		For example:
+		[codeblock]
+		func parse_file(path, extracted_strings):
+		    var res = ResourceLoader.load(path, "Script")
+		    var text = res.get_source_code()
+		    # Parsing logic
+
```

Comment:
A period at the end of comment and a extra line here too.

Reply:
Right! Forgot everytime.